x-pack/legacy/plugins/siem/cypress/integration/smoke_tests/inspect/inspect.spec.ts

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { logout } from '../../lib/logout';
+import { HOSTS_PAGE } from '../../lib/urls';
+import {
+  INSPECT_BUTTON_ICON,
+  INSPECT_MODAL,
+  INSPECT_BUTTONS_IN_SIEM,
+  TIMELINE_SETTINGS_ICON,
+  TIMELINE_INSPECT_BUTTON,
+} from '../../lib/inspect/selectors';
+import { DEFAULT_TIMEOUT, loginAndWaitForPage } from '../../lib/util/helpers';
+import { executeKQL, hostExistsQuery, toggleTimelineVisibility } from '../../lib/timeline/helpers';
+describe('Inspect', () => {
```

Comment:
nit: add some newlines throughout this spec?

Reply:
boo newlines... ill consider it ;)

Reply:
when i autolint these files, this is what happens. im leaving it! 😜